packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -3,7 +3,9 @@ import { generateTestClient } from '../../../../utils/getTestClient'
 import type { SetupParams } from '../../../../utils/setupMSSQL'
 import { setupMSSQL } from '../../../../utils/setupMSSQL'
 
-describe('blog-env-mssql', () => {
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
```

Comment:
Uh, no nicer/built-in way to do this in Jest?

Reply:
Not that I know of, unfortunately 😞
I would be very glad if someone has better suggestions, but I'm not too optimistic about that since I basically repeated the `testIf` pattern which was already used in the codebase.